.vsts-ci/windows.yml

```diff
@@ -90,8 +90,8 @@ jobs:
 
   - powershell: |
       Import-Module .\tools\ci.psm1
-      $ParallelXUnitTestResultsFile = "$(System.ArtifactsDirectory)\xunit\ParallelXUnitTestResults.xml"
+      $xUnitTestResultsFile = "$(System.ArtifactsDirectory)\xunit\xUnitTestResults.xml"
```

Comment:
We could do something smarter here.  Like:

1. Get all items in the `xunit` folder
2. run `Test-XUnitTestResults` on each item found
3. or throw an exception if no items were found

Reply:
Although, this is not changed frequently.  So, this is not blocking this PR.